src/db/models/article.py

```diff
@@ -14,3 +15,10 @@ class Article(Base):
     link = Column(String())
```

Comment:
Jak już mamy tabele na linki, to nie trzeba odzielnego pola na link w modelu `Article`  trzymać, można też foreign_key do tablicy z linkami.
```python
article_link_id = Column(Integer, ForeignKey('links.id'))
```